components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -3568,6 +3568,11 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         return [];
     }
 
+    async reportErrorBoundary(ctx: TraceContextWithSpan, url: string, message: string): Promise<void> {
+        log.info("dashboard error boundary", { message, url, userId: this.user?.id });
```

Comment:
Wasn't sure if this should be a `log.error()` or `log.info()` so played it safe. Happy to adjust though.

Reply:
Would be good to limit the size of the message and/or error as these contribute directly to our logging cost